### PR TITLE
Add polar stereo

### PR DESCRIPTION
**Pull request by mjhossen**
## Description

Modified the code ```figure_driver.py``` to enable us to plot figures over polar region. 

## Dependencies

If there are PRs that need to be merged before or along with this one, please list them below
Waiting on the following PRs:
- [ ] waiting on EMCpy https://github.com/NOAA-EMC/emcpy/pull/109

### CODE REVIEW

**Comment by mjhossen** [comment]
Plot the following figures using ```testIodaObsSpaceAmsuaN19.yaml```

Screen Shot 2022-11-02 at 11 31 49 AM

**Comment by danholdaway** [comment]
Once the EMCpy change is merged you will also need to bump the hash here: https://github.com/JCSDA-internal/eva/blob/5d47296dbc26a5a0e320753bc8a6646a4c6a531a/setup.py#L43

**Comment by danholdaway** [comment]
The test is failing because it doesn't pick up the correct EMCpy. It should pass once the hash is correct. But please don't add a completely new test. Just make one additional plot using the polar stereo projection in the existing AMSU N19 YAML file.

**Comment by CoryMartin-NOAA** [comment]
@mjhossen as @danholdaway said now that emcpy is updated, update the hash to: `4f36baf1a2302fb0daa49bd8415bb7d2a65347bb`

**Comment by danholdaway** [comment]
Tests are passing now. Please modify the tests so as not to add an entirely new test.

File: setup.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
         'scikit-learn>=1.0.2',
         'xarray>=0.11.3',
         'emcpy @ git+https://github.com/NOAA-EMC/' +
-        'emcpy@9e55f5c64a03dbca56ff9425a480d6906e834e17#egg=emcpy',
+        'emcpy@3ff4eeb63ed341d79483e18265c1de75c810246f#egg=emcpy',
```
Comment (CoryMartin-NOAA) [inline setup.py]: What commit is this? It doesn't look like the current head of develop?

**Review by CoryMartin-NOAA — CHANGES_REQUESTED** [review]
"This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."

**Review by CoryMartin-NOAA — APPROVED** [review]

**Pull request merged**